string.go

```diff
@@ -37,12 +38,39 @@ func RandomString(size int, charset []rune) string {
 		panic("lo.RandomString: Charset parameter must not be empty")
 	}
 
-	b := make([]rune, size)
-	possibleCharactersCount := len(charset)
-	for i := range b {
-		b[i] = charset[rand.IntN(possibleCharactersCount)]
+	sb := strings.Builder{}
+	sb.Grow(size)
+	letterIdBits := int(math.Log2(float64(nearestPowerOfTwo(len(charset)))))
+	var letterIdMask int64 = 1<<letterIdBits - 1
+	letterIdMax := 63 / letterIdBits
```

Comment:
Why do we only use 63 bits for int64? This is because the first bit indicates the signpositive or negative), and it is not random
```go
// Int64 returns a non-negative pseudo-random 63-bit integer as an int64
// from the default Source.
func Int64() int64 { return globalRand.Int64() }
```